docs/troubleshooting.md

```diff
@@ -0,0 +1,14 @@
+# Troubleshooting Guide
+
+## Troubleshooting AKS cluster creation
+
+### 1. AKS cluster creation fails
+
+If you notice the following error message while the cluster is being provisioned, it could indicate the Azure authentication token has expired.
```

Comment:
```suggestion
If you notice the following error message from `az aks create`, it could indicate the Azure authentication token has expired.
```

(to make it clear it is not from `kubeprod install aks`)